ci-operator/step-registry/cucushift/upgrade/mirror-images/by-oc-adm/cucushift-upgrade-mirror-images-by-oc-adm-commands.sh

```diff
@@ -54,13 +64,6 @@ if [[ -f "${SHARED_DIR}/kubeconfig" ]] ; then
     export KUBECONFIG=${SHARED_DIR}/kubeconfig
 fi
 
-# Setup proxy if it's present in the shared dir
```

Comment:
Why move it to `update_icsp`, I think `mirror_image` also need the proxy when `--apply-release-image-signature` enabled.

Reply:
@jianlinliu From @shellyyang1989 's comment on https://github.com/openshift/release/pull/37250, it seems there are no other function need it, if I am wrong please correct me.

```
The test failed on upgrade target mirror step because we set up proxy in the beginning of the step which doesn't have access to internet. But seems like only function update_icsp() requires cluster access. So we can fix it by moving https://github.com/openshift/release/blob/master/ci-operator/step-registry/cucushift/upgrade/mirror-images/by-oc-adm/cucushift-upgrade-mirror-images-by-oc-adm-commands.sh#L57-L62 to https://github.com/openshift/release/blob/master/ci-operator/step-registry/cucushift/upgrade/mirror-images/by-oc-adm/cucushift-upgrade-mirror-images-by-oc-adm-commands.sh#L38 and unsetting proxy at the end of the function.
```

Reply:
For pure mirror command, doesn't need the proxy, unfortunately once enabled `--apply-release-image-signature`, it will try to connect the cluster to create some resource, once that need proxy to access cluster.

Reply:
This is the WIP PR:
https://github.com/openshift/release/pull/38236